.golangci.yaml

```diff
@@ -0,0 +1,131 @@
+run:
+  tests: true
+  timeout: 5m
+  skip-dirs:
+    - resources
+    - old
+  skip-files:
+    - cmd/protopkg/main.go
+
+output:
+  print-issued-lines: false
+
+linters:
+  enable-all: true
+  disable:
```

Comment:
we hate linters lol

Reply:
lol. copied directly from [here](https://github.com/TBD54566975/ftl/blob/main/.golangci.yml)